react/src/components/ReviewTile.js

```diff
@@ -1,16 +1,75 @@
 import React from 'react'
 import { Link } from 'react-router';
 
-const ReviewTile = (props) => {
+class ReviewTile extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      vote_count: this.props.vote_count
+    }
+    this.handleUpvote = this.handleUpvote.bind(this);
+    this.handleDownvote = this.handleDownvote.bind(this);
+  }
+
+  handleUpvote() {
+    let value = this.state.vote_count += 1;
+    let id = this.props.id;
+    this.setState({ vote_count: value });
+    let votePayload = {
+      review_id: id,
+      overall_rating: this.props.overall_rating,
+      user_id: this.props.user_id,
+      site_id: this.props.site_id,
+      design_body: this.props.design_body,
+      usability_body: this.props.usability_body,
+      concept_body: this.props.concept_body,
+      vote_count: this.state.vote_count
+    }
+    this.sendVote(votePayload);
+  }
+
```

Comment:
We weren't able to get the data to persist when only sending the vote_count, so we sent all of the params, even though the user isn't able to edit them.